src/frontend/planner_test/tests/testdata/agg.yaml

```diff
@@ -701,18 +701,15 @@
     create table t(a int, b int, c int);
     select a, count(distinct b) as distinct_b_num, sum(distinct c) filter(where c < 100) as distinct_c_sum from t group by a;
   optimized_logical_plan: |
-    LogicalAgg { group_key: [t.a], aggs: [count(t.b) filter((flag = 0:Int64)), sum(t.c) filter((count filter((t.c < 100:Int32)) > 0:Int64) AND (flag = 1:Int64))] }
-    └─LogicalAgg { group_key: [t.a, t.b, t.c, flag], aggs: [count filter((t.c < 100:Int32))] }
-      └─LogicalExpand { column_subsets: [[t.a, t.b], [t.a, t.c]] }
-        └─LogicalScan { table: t, columns: [t.a, t.b, t.c] }
```

Comment:
Hmm, is this only because we do 2-phase-agg for stream?

Reply:
Ditto.